definitions/json/ComplexTypes/ComplexTypes.json

```
@@ -3860,16 +3860,15 @@
     "ID": "DraftAndSignJudgement",
     "ListElementCode": "isJudgement",
     "FieldType": "YesOrNo",
-    "ElementLabel": "Is this a Judgement",
+    "ElementLabel": "Is this a Judgment",
     "SecurityClassification": "Public"
   },
   {
     "ID": "DraftAndSignJudgement",
-    "ListElementCode": "draftAndSignJudgementDocument",
-    "FieldType": "Document",
-    "ElementLabel": "Document upload",
+    "ListElementCode": "DraftAndSignJudgementDocuments",
```

just to make it consistent
```suggestion
    "ListElementCode": "draftAndSignJudgementDocuments",
```